src/models/propulsion/FGTurbine.h

```diff
@@ -337,6 +341,34 @@ class FGSpoolUp : public FGParameter
   FGTurbine* turb;
   double delay; ///< Inverse spool-up time from idle to 100% (seconds)
 };
+
+class FGTSFC : public FGParameter
+{
+public:
+  FGTSFC(FGTurbine* _turb, double tsfcVal)
+    : turb(_turb), tsfc(new FGRealValue(tsfcVal)) {}
+
+  FGTSFC(FGTurbine* _turb, FGFDMExec* fdmexec, Element* element, int enginenumber)
+    : turb(_turb), tsfc(new FGFunction(fdmexec, element, to_string(enginenumber))) {}
+
+  string GetName(void) const { return string(); }
+
+  double GetValue(void) const {
+    // If there is a user supplied function don't perform any correction/denormalisation
+    if(dynamic_cast<FGFunction*>(tsfc.get()))
+      return tsfc->GetValue();
+    else {
+      double T = turb->in.Temperature;
+      double N2norm = turb->N2norm;
+      return tsfc->GetValue() * sqrt(T / 389.7) * (0.84 + (1 - N2norm) * (1 - N2norm));
+    }
```

Comment:
Just manage the case where TSFC is not specified as a function:
```suggestion
    double T = turb->in.Temperature;
    double N2norm = turb->N2norm;
    return tsfc * sqrt(T / 389.7) * (0.84 + (1 - N2norm) * (1 - N2norm));
```